mon-pix/translations/fr.json

```diff
@@ -176,7 +176,12 @@
 
     "error": {
       "content-text": "'<p>'Nous vous invitons à recharger cette page ou '<LinkTo @route=\"login\">'revenir à la page d’accueil'</LinkTo>'.'</p><p>'Vous pouvez aussi nous contacter via '<a href=\"https://support.pix.fr/support/tickets/new\">'le formulaire du centre d'aide'</a>' en nous précisant le code d’erreur ci-dessous dans la description.'</p><p>'Nous vous prions de nous excuser pour la gêne occasionnée.'</p><p>'L’équipe Pix.'</p>'",
-      "first-title": "Oups, une erreur est survenue !"
+      "first-title": "Oups, une erreur est survenue !",
+      "campaign": {
+        "forbidden": "Oups ! nous ne parvenons pas à vous trouver. Vérifiez vos informations afin de continuer ou prévenez l’organisateur.",
+        "not-found": "Votre code est erroné, veuillez vérifier ou contacter l’organisateur.",
+        "code-missing": "Veuillez saisir un code."
```

Comment:
Ces erreurs font parties de la page `fill-in-campaign-code`, vu qu'on ne change pas de page pour arriver sur la page `error`.
Je propose de les déplacer dans une partie `errors` sous `pages.fill-in-campaign-code`

Reply:
Je me posais la question de leur place,
Je suis d'accord dans `page.errors` c'est pas l'idéal. J'ai l'impression que tous ces messages d'erreurs devraient être regroupés autrement ?
Si on les groupe par page on risque de dupliquer des traductions ?

Reply:
Les textes ici sont très contextuelle au démarrage de la campagne (saisir un code, contacter l'organisateur, etc...), ce ne sont donc pas des erreurs classiques, mais surtout elle n'apparaisse que dans une seule page actuellement.

Reply:
D'acc ! C'est fait !